ui/app/src/state/actions/pathNavigator.ts

```diff
@@ -73,8 +73,11 @@ export const pathNavigatorFetchParentItems = /*#__PURE__*/ createAction<
   PayloadWithId<{ path: string; excludes?: string[]; limit: number; offset?: number; keyword?: string }>
 >('PATH_NAVIGATOR_FETCH_PARENT_ITEMS');
 
+export const pathNavigatorFetchPath =
+  /*#__PURE__*/ createAction<PayloadWithId<{ path: string }>>('PATH_NAVIGATOR_FETCH_PATH');
+
 export const pathNavigatorFetchPathComplete = /*#__PURE__*/ createAction<
-  PayloadWithId<{ parent?: DetailedItem; children: GetChildrenResponse }>
+  PayloadWithId<{ parent?: DetailedItem; children: GetChildrenResponse; path?: string }>
```

Comment:
What was path required for? Can't you get the path from `payload.parent`. It being conditional is not great so rather remove.

Reply:
I was able to get the path from `parent`. The only detail is that in `pathNavigatorChangePage` epic it wasn't sending the parent, but I was able to retrieve it from path navigator currentPath